miniapp/miniapp_triangular_solver.cpp

```diff
@@ -109,7 +110,7 @@ int hpx_main(hpx::program_options::variables_map& vm) {
     sync_barrier();
 
     dlaf::common::Timer<> timeit;
-    dlaf::solver::triangular<Backend::MC>(comm_grid, side, uplo, op, diag, alpha, a, b);
+    dlaf::solver::triangular<Backend::MC, Device::CPU, T>(comm_grid, side, uplo, op, diag, alpha, a, b);
```

Comment:
Just curious: are the added template parameters required or just for explicitness?

Reply:
For explicitness, but I can revert if you prefer.

Reply:
No preference, I'm happy to keep your change.